src/fsharp/vs/service.fs

```diff
@@ -2442,16 +2434,12 @@ type FSharpProjectFileInfo (fsprojFileName:string, ?properties, ?enableLogging)
               ]
 
         let references =
-              [  for i in project.GetItems("ReferencePath") do
-                   yield i.EvaluatedInclude
-                 for fsproj in projectReferences do
-                   match (try let p' = projectInstanceFromFullPath fsproj
-                              Some (outFileOpt p') with _ -> None) with
-                   | Some (Some output) -> yield output
-                   | _ -> ()
-              ]
+              [ for i in project.GetItems("ReferencePath") do
+                  yield i.EvaluatedInclude
+                for i in project.GetItems("ChildProjectReferences") do
```

Comment:
"projectReferences" contains paths to the fsproj files referenced, while "ChildProjectReferences" contains the output assemblies.